integration_tests/dbt_project.yml

```diff
@@ -23,6 +23,7 @@ vars:
     bundle:                     "{{ ref('bundle_data') }}"
     credit_memo:                "{{ ref('credit_memo_data') }}"
     credit_memo_line:           "{{ ref('credit_memo_line_data') }}"
+    credit_card_payment_txn: "{{ ref('credit_card_payment_txn_data') }}"
```

Comment:
Minor, but should we make sure the `ref` statement is in line with the rest of the identifiers for style purposes?

Reply:
Sure, just updated!

Reply:
Sure! Just updated